.github/workflows/docker_build_kanidm.yml

```diff
@@ -78,4 +87,4 @@ jobs:
           echo "${{ secrets.GITHUB_TOKEN }}" | \
             oras login -u "${{ github.actor }}" --password-stdin ghcr.io
           oras copy --from-oci-layout "/tmp/kanidm-docker.tar:devel" \
-            "ghcr.io/${{ github.repository_owner }}/kanidm:devel"
+            "ghcr.io/${{ needs.set_tag_values.outputs.owner_lc }}/kanidm:devel"
```

Comment:
These are guaranteed to be the same?

Reply:
No they're not, `repository_owner` can have capitals - in fact your account would fail to build this without the lower-casing that the step does to it.

Reply:
So they are not the same but in a good way. Love this change ❤️